src/Shmuelie.WinRTServer/Shmuelie.WinRTServer.csproj

```diff
@@ -2,22 +2,16 @@
   <PropertyGroup>
     <TargetPlatformVersion>10.0.22000.0</TargetPlatformVersion>
```

Comment:
Don't set this, just put the Windows SDK in the TFM. The .NET SDK will set this for you.